server/public/styles.css

```diff
@@ -213,9 +224,10 @@ i {
 }
 
 .option-paper.is-showing {
-  animation: show-paper 1s 0.25s linear 1;
+  animation: show-paper 1s linear 1;
   animation-fill-mode: forwards;
-  transform: translateX(0.25rem) translateY(0.25rem);
+
+  /* transform: translateX(0.25rem) translateY(0.25rem); */
```

Comment:
please remove

Reply:
Removed. Missed the one 

![image](https://user-images.githubusercontent.com/109567499/201221856-9d8af80b-59d1-432a-b496-e0d62540a808.png)